src/manager.cc

```diff
@@ -334,8 +336,36 @@ fastcat::Manager::GetDeviceStatePointers()
 }
 
 double fastcat::Manager::GetTargetLoopRate() { return target_loop_rate_hz_; }
+
 bool   fastcat::Manager::IsFaulted() { return faulted_; }
 
+void fastcat::Manager::InitializeActuatorNames() {
+  actuator_names_.clear();
+  for (auto& device : jsd_device_list_) {
+    if(device->GetState()->type == ACTUATOR_STATE) {
+      actuator_names_.push_back(device->GetName()); 
+    }
+  }
+}
+
+const std::vector<std::string>& fastcat::Manager::GetActuatorNames() {
+  return actuator_names_;
+}
+
+bool fastcat::Manager::GetActuatorParams(
+  const std::string& name, fastcat::Actuator::ActuatorParams& params)
+{
+  if(device_map_.count(name)) {
+    auto& device = device_map_[name];
+    if(device->GetState()->type == ACTUATOR_STATE) {
+      auto actuator = std::dynamic_pointer_cast<Actuator>(device);
+      params = actuator->GetParams();
+      return true;
+    }
+  }
+  return false;
+}
+
```

Comment:
This is fine for now.  In the (hopefully) near future, I'll make this more generalizable for all device types with something something like:
``` c++
bool fastcat::Manager::GetDeviceParams(const std::string& name, fastcat::device_config& params)
```
Where the notional `device_config` would be a auto-coded union. Just an FYI that API break may come in the future.